demisto_sdk/commands/validate/validators/PB_validators/PB100_is_no_rolename.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    FixResult,
+    ValidationResult,
+)
+
+ContentTypes = Playbook
+
+
+class IsNoRolenameValidator(BaseValidator[ContentTypes]):
+    error_code = "PB100"
+    description = "Validate whether the playbook has a rolename. If the Playbook has a rolename it is not valid."
+    error_message = "The playbook '{playbook_name}' can not have a rolename, please remove the field."
+    fix_message = (
+        "Removed the 'rolename' from the following playbook '{playbook_name}'."
+    )
+    related_field = "rolename"
+    is_auto_fixable = True
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        return [
+            ValidationResult(
+                validator=self,
+                message=self.error_message.format(
+                    playbook_name=content_item.name,
+                ),
+                content_object=content_item,
+            )
+            for content_item in content_items
+            if content_item.data.get("rolename", None)
```

Comment:
Let add rolename as a classVar of the base_playbook object rather than read it from the data.

Reply:
But If the Playbook has a rolename it is not valid. So maybe better not to add?

Reply:
Good point, but then if it won't be a classVar then the fix won't really work..
So it's either we remove the fix for this validation or we add this classVar, I'm fine with both cases.. up to you to decide.